kitsune/groups/templatetags/jinja_helpers.py

```diff
@@ -25,3 +27,13 @@ def group_link(group):
         return Markup(html)
     except GroupProfile.DoesNotExist:
         return group.name
+
+
+@library.global_function
+def private_message_group(group):
```

Comment:
As far as I can tell this is only used in the `profile.html`. If this is the case, why don't we add the html directly into the profile?

Even if we wanted to extract this, probably a macro would be a better candidate. For now I would suggest to just add it in the html and extract it only when/if needed.